project/project_meta.go

```diff
@@ -0,0 +1,43 @@
+package project
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+
+	"github.com/pkg/errors"
+
+	"github.com/machinefi/sprout/types"
+)
+
+type ProjectMeta struct {
+	ProjectID uint64
+	Uri       string
+	Hash      [32]byte
+	Paused    bool
+}
+
+func (m *ProjectMeta) ProjectConfig() (*Config, error) {
+	rsp, err := http.Get(m.Uri)
```

Comment:
maybe need add fetch file from ipfs logic

Reply:
`ioctl` uploads resource to IPFS and generates a `http` URL for resource fetching